R/cmapplot.R

```diff
@@ -42,7 +42,7 @@ if (.Platform$OS.type == "windows") {
       font_sbold = "Whitney Semibold"
     )
   } else {
-    warning("Whitney is not installed on this PC, so CMAP theme will default to Calibri")
+    message("Whitney is not installed on this PC, so CMAP theme will default to Calibri")

```

Comment:
I changed these from `warning` to `message`. 
Motivation: I don't like how warning messages take two lines.
Justification: we're not [fundamentally worried about how the package will function](https://www.r-bloggers.com/a-warning-about-warning/) if this error is produced, nor do we want the package to error in an environment with a [heightened warning setting](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/warning).

Reply:
Can we change it to `message("WARNING: Whitney is not ...` in that case? I do want users to feel like they are being warned rather than just brushing these massages off.

Reply:
Works for me.